themes/bootstrap3/js/covers.js

```diff
@@ -10,6 +10,12 @@ function loadCoverByElement(data, element) {
     if (typeof response.data.url !== 'undefined' && response.data.url !== false) {
       img.attr("src", response.data.url);
       container.children().not("img").hide();
+      if (typeof response.data.backlink_text !== 'undefined' && typeof response.data.backlink_url !== 'undefined') {
```

Comment:
Should we also support the case where a credit is desired but it has no associated link? (i.e. backlink_text without backlink_url)?

Reply:
I've done it, thanks for the idea.